options.c

```diff
@@ -5064,7 +5064,7 @@ static char *fio_keyword_replace(char *opt)
 		struct fio_keyword *kw = &fio_keywords[i];
```

Comment:
Maybe the area in the git commit title should be `options: ` ?

Reply:
ok